mixer/testdata/config/cloudwatch.yaml

```diff
@@ -0,0 +1,39 @@
+# instance configuration for template 'metric'
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+ name: cloudwatchrequestcount
+ namespace: istio-system
+spec:
+ value: "1"
+ dimensions:
+   target: destination.service | "unknown"
+   service: destination.labels["app"] | "unknown"
```

Comment:
curious: in this config, what is the expected semantic difference between `target` and `service` ?